deliver/lib/deliver/options.rb

```diff
@@ -274,7 +274,17 @@ def self.available_options
         FastlaneCore::ConfigItem.new(key: :marketing_url,
                                      description: "Metadata: Localised marketing url",
                                      optional: true,
-                                     is_string: false)
+                                     is_string: false),
+        FastlaneCore::ConfigItem.new(key: :languages,
+                                     description: "Metadata: List of languages to activate",
+                                     type: Array,
+                                     optional: true,
+                                     verify_block: proc do |value|
+                                       value.each do |language|
+                                         is_valid_language = FastlaneCore::Languages::ALL_LANGUAGES.include?(language)
+                                         UI.user_error!(":languages was given an invalid lanuage - #{language}") unless is_valid_language
```

Comment:
* typo `lanuage`
* also I am not sure we expose the symbol syntax to the user in the error message. Maybe remove the `:`
* if more than one invalid, would it be more logical to return all of the invalid at once?